guides/v2.2/cloud/project/project-upgrade.md

```diff
@@ -155,7 +155,10 @@ For an upgrade, you delete the `config.php` file. Once this file is added to you
 
 ## Verify and upgrade your extensions {#extensions}
 
-If you need to upgrade any third-party extensions and modules that support version 2.2, we recommend working in a new Integration branch with your extensions disabled. Review your third-party extension and module pages in Marketplace or other company sites to verify support for {{site.data.var.ee}} and {{site.data.var.ece}} version 2.2.
+{: .bs-callout-info }
+Review your third-party extension and module pages in Marketplace or other company sites to verify support for {{site.data.var.ee}} and {{site.data.var.ece}} version 2.2.
```

Comment:
Multi-line notes require different formatting to appear properly:

```
{%
include note.html
type='info'
content='Review your third-party extension and module pages in Marketplace or other company sites to verify support for {{site.data.var.ee}} and {{site.data.var.ece}} version 2.2.
  
If you need to upgrade any third-party extensions and modules that support version 2.2, we recommend working in a new Integration branch with your extensions disabled.'
%}
```

Reply:
**implemented**

Reply:
@jeff-matthews , PR is waiting for you ;)

Thanks